app/views/articles/show.html.erb

```diff
@@ -9,14 +10,16 @@
   <% else %>
     <% @follows.each do |follow| %>
       <% if follow.id == @article.user_id || @article.user_id == current_user.id  %>
-        <div style = "margin-left: 85%;padding: 1rem; max-width: 50rem;">
-          <!-- Button trigger modal -->
-          <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal">Add comment</button>
-        </div>
+        <% cont = 1 %>
```

Comment:
which is the use of the `cont` variable?

Reply:
I have used this counter due to the fact that when a user reviews the profile of another I bring all the users that this follows and thus I validate that this user is in the list of followers and can comment on an article

Reply:
Could it be done with RoR methods?